3rd-party/class.jetpack-amp-support.php

```diff
@@ -12,6 +12,12 @@ static function init() {
 			return;
 		}
 
+		/**
+		 * Remove this during the init hook in case users have enabled it during
+		 * the after_setup_theme hook, which triggers before init.
+		 */
+		remove_theme_support( 'jetpack-devicepx' );
```

Comment:
I think the AMP module should be responsible for removing the devicepx support to ensure that API changes in the AMP module don't have to be updated across other modules. Secondly, we're 99.9% sure that users don't want devicepx on AMP so we run it during `init` which is after `after_setup_theme`.